inc/session.class.php

```diff
@@ -719,11 +719,21 @@ static function getPluralNumber() {
    **/
    static function isCron() {
 
-      return (isset($_SESSION["glpicronuserrunning"])
+      return (self::isInventory() || isset($_SESSION["glpicronuserrunning"])
```

Comment:
Are you sure you want to consider all inventory requests as crontasks ?

Reply:
Since the isInventory is conditioned on the URL anyways; yes, all existing isCron should just work fine